src/hooks/use-toast.ts

```diff
@@ -13,7 +13,8 @@ const TOAST_REMOVE_DELAY = 1000000
 
 type ToasterToast = ToastProps & {
   id: string
-  title?: React.ReactNode
+  path?: string
+  isSuccess?: Boolean
```

Comment:
@nadirhussain786 `isSuccess` is mandatory

Reply:
Fixed